packages/angular/src/generators/scam/schema.d.ts

```diff
@@ -0,0 +1,18 @@
+export interface Schema {
+  name: string;
+  path?: string;
+  project?: string;
+  displayBlock?: boolean;
+  inlineStyle?: boolean;
+  inlineTemplate?: boolean;
+  viewEncapsulation?: 'Emulated' | 'None' | 'ShadowDom';
+  changeDetection?: 'Default' | 'OnPush';
+  style?: 'css' | 'scss' | 'sass' | 'less' | 'none';
+  skipTests?: boolean;
+  inlineScam?: boolean;
+  type?: string;
+  flat?: boolean;
+  prefix?: string;
```

Comment:
I can't find `prefix` defined in the `schema.json`.

Reply:
ah good catch